doc/pages/contributors.json

```diff
@@ -254,5 +254,9 @@
   {
     "name": "Annette Alt",
     "link": "https://homestylecookingwithsylvya.wordpress.com/"
+  },
+  {
+    "name": "Jasper Martin",
```

Comment:
You actually don't need to put your name here if you filled out the blurb part!

Reply:
Oh sorry about that